webhook/server/server.go

```diff
@@ -75,10 +79,16 @@ func registerInjectEndpoint(mgr manager.Manager, ns string, podName string) erro
 		return err
 	}
 
+	var UID types.UID
+	if UID, err = kubesystem.GetUID(mgr.GetAPIReader()); err != nil {
```

Comment:
```suggestion
	if UID, err = kubesystem.GetUID(mgr.GetClient()); err != nil {
```

In general prefer `.GetClient()` over `.GetAPIReader()` - the former has a cache and should be used when possible (i.e., basically all cases except when accessing objects in foreign namespaces.)

Not that important for this case in particular, since it's an one-time operation.